addd Highlights for 2023-03-23 (#1735)
giantswarm/docs: src/content/changes/highlights/2023-03-23.md

---
date: 2023-03-23T11:00:00
title: Highlights for the week ending March 23, 2023
changes_categories:
  - Highlights
owner:
  - https://github.com/orgs/giantswarm/teams/sig-product
---
### Observability

> - Prometheus-agent: New cluster releases contain a new component which enables us to monitor your cluster and provides the best quality of service. Some cluster upgrades might require manual intervention from our side, especially if you are using Prometheus Operator. Feel free to ask our team for any questions or help needed (@support-atlas). In case you want to know more details about why we introduced such a change please read our [documentation](https://docs.giantswarm.io/getting-started/observability/monitoring/prometheus/agent/).

Reviewer: ```suggestion
- [Prometheus-agent](https://github.com/giantswarm/prometheus-agent-app): New cluster releases contain a new component which enables us to monitor your cluster and provides the best quality of service. Some cluster upgrades might require manual intervention from our side, especially if you are using Prometheus Operator. Feel free to ask our team for any questions or help needed (@support-atlas). In case you want to know more details about why we introduced such a change please read our [documentation](https://docs.giantswarm.io/getting-started/observability/monitoring/prometheus/agent/).
```

> - Grafana Explore: It is a new feature enabled in your Grafana installation. This allows you to execute PromQL (and LogQL later) queries against metrics we collect from your clusters. See [our docs for more information](https://docs.giantswarm.io/getting-started/observability/grafana/access/ https://grafana.com/docs/grafana/latest/explore/)

Reviewer: ```suggestion
- Grafana Explore: It is a new feature enabled in your Grafana installation. This allows you to execute PromQL (and LogQL later) queries against metrics we collect from your clusters. See [our docs for more information](https://docs.giantswarm.io/getting-started/observability/grafana/access/)
```
Link was broken. @TheoBrigitte Is there a grafana release that can we link here?

* prometheus-agent: we are in the process of deploying a new component to workload cluster which enables us to monitor your cluster and provides best quality of service. Some cluster upgrade might require manual intervention from our side, especially if you are using prometheus-operator. Feel free to ask team-atlas for any questions or help needed, also more details like which releases are affected can be found in our doc [here](https://docs.giantswarm.io/getting-started/observability/monitoring/prometheus/agent/).
* grafana explore: we opened access to Grafana Explore feature which allow you to execute PromQL (and LogQL later) queries against all our metrics. See https://docs.giantswarm.io/getting-started/observability/grafana/access/ https://grafana.com/docs/grafana/latest/explore/
* vertical-pod-autoscaler: v3.x is out which might break with your existing config. Also v3.2.0 might help regarding resources usage as VPA now autoscales itself. Fee free to ask team-atlas for any questions or help needed. See [github.com/vertical-pod-autoscaler](https://github.com/giantswarm/vertical-pod-autoscaler-app/releases/tag/v3.0.0)

> ## Apps
> 

Reviewer 2: ```suggestion
## Apps

- [kyverno-app](https://github.com/giantswarm/kyverno-app) version [v0.14.1](https://github.com/giantswarm/kyverno-app/blob/main/CHANGELOG.md#0141---2023-03-22) updates to Kyverno version 1.9.2 and policy-reporter version 2.17.0. This version includes performance improvements to increase the AdmissionReports processing speed and reduce the number of reports stored in the cluster.
- [trivy-app](https://github.com/giantswarm/trivy-app/) version [0.8.0](https://github.com/giantswarm/trivy-app/blob/main/CHANGELOG.md#080---2023-03-17) updates the Trivy scanner version to v0.37.2, containing various bug fixes and additional support for future scanning features.

```

> - Vertical-Pod-Autoscaler (VPA): New version `v.3.x` is our with new improvements. Highlight that now VPA auto-scales itself. This version requires updating the existing config. Read the [migration docs](https://github.com/giantswarm/vertical-pod-autoscaler-app#moving-from-2xx-to-3xx---breaking-change) and feel free to ask our team in case of any doubt (@support-atlas).

Reviewer: ```suggestion
- [Vertical-Pod-Autoscaler (VPA)](https://github.com/giantswarm/vertical-pod-autoscaler-app): New version [`v.3.x`](https://github.com/giantswarm/vertical-pod-autoscaler-app/releases/tag/v3.3.0) is our with new improvements. Highlight that now VPA auto-scales itself. This version requires updating the existing config. Read the [migration docs](https://github.com/giantswarm/vertical-pod-autoscaler-app#moving-from-2xx-to-3xx---breaking-change) and feel free to ask our team in case of any doubt (@support-atlas).
```

> - [external-dns-app](https://github.com/giantswarm/external-dns-app) version [2.34.1](https://github.com/giantswarm/external-dns-app/blob/master/CHANGELOG.md#2341---2023-03-22) adds `node-role.kubernetes.io/control-plane` to the toleration of crd install jobs. This update also contains the addition of `ServiceMonitor` and the addition of default values that were released in version [2.34.0](https://github.com/giantswarm/external-dns-app/blob/master/CHANGELOG.md#2340---2023-03-21)

Reviewer: ```suggestion

- [External-dns-app](https://github.com/giantswarm/external-dns-app) version [2.34.1](https://github.com/giantswarm/external-dns-app/blob/master/CHANGELOG.md#2341---2023-03-22) adds `node-role.kubernetes.io/control-plane` to the toleration of CRD install jobs. This update also contains the addition of `ServiceMonitor` and the addition of default values that were released in version [2.34.0](https://github.com/giantswarm/external-dns-app/blob/master/CHANGELOG.md#2340---2023-03-21)
```

> - [kyverno-app](https://github.com/giantswarm/kyverno-app) version [v0.14.1](https://github.com/giantswarm/kyverno-app/blob/main/CHANGELOG.md#0141---2023-03-22) updates to Kyverno version 1.9.2 and policy-reporter version 2.17.0. This version includes performance improvements to increase the AdmissionReports processing speed and reduce the number of reports stored in the cluster.
> - [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) version [2.27.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/main/CHANGELOG.md#2270---2023-03-22)This is the second part of our upstream alignement. To be aligned with upstream we added 7 helper properties, 43 deployment properties and added CI values from upstream. Please take care and read the [changelog](https://github.com/giantswarm/nginx-ingress-controller-app/blob/main/CHANGELOG.md#2270---2023-03-22) as there are some changes that should be done with care. But not only properties got added, in the process of alignement we also changed 9 helpers and 5 deployment properties. Additionally the chart.yaml got changed to be aligned with upstream. Last but not least we removed 2 helper and 9 deployment properties. Please be aware that some of these changes, additons and removals can require some changes in our delpoyments. Please read the [changelog](https://github.com/giantswarm/nginx-ingress-controller-app/blob/main/CHANGELOG.md#2270---2023-03-22) for all information. 
> - [trivy-app](https://github.com/giantswarm/trivy-app/) version [0.8.0](https://github.com/giantswarm/trivy-app/blob/main/CHANGELOG.md#080---2023-03-17) updates the Trivy scanner version to v0.37.2, containing various bug fixes and additional support for future scanning features.

Reviewer: ```suggestion
- [Kyverno-app](https://github.com/giantswarm/kyverno-app) version [v0.14.1](https://github.com/giantswarm/kyverno-app/blob/main/CHANGELOG.md#0141---2023-03-22) updates to Kyverno version 1.9.2 and policy-reporter version 2.17.0. This version includes performance improvements to increase the AdmissionReports processing speed and reduce the number of reports stored in the cluster.
- [Nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) version [2.27.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/main/CHANGELOG.md#2270---2023-03-22)This is the second part of our upstream alignement. To be aligned with upstream we added 7 helper properties, 43 deployment properties and added CI values from upstream. Please take care and read the [changelog](https://github.com/giantswarm/nginx-ingress-controller-app/blob/main/CHANGELOG.md#2270---2023-03-22) as there are some changes that should be done with care. But not only properties got added, in the process of alignment, we also changed 9 helpers and 5 deployment properties. Additionally, the `chart.yaml` got changed to be aligned with upstream. Last but not least we removed 2 helper and 9 deployment properties. Please be aware that some of these changes, additions and removals can require some changes in our deployments. Please read the [changelog](https://github.com/giantswarm/nginx-ingress-controller-app/blob/main/CHANGELOG.md#2270---2023-03-22) for all information. 
- [Trivy-app](https://github.com/giantswarm/trivy-app/) version [0.8.0](https://github.com/giantswarm/trivy-app/blob/main/CHANGELOG.md#080---2023-03-17) updates the Trivy scanner version to `v0.37.2`, containing various bug fixes and additional support for future scanning features.
```

Reviewer 2: FWIW I keep the name the actual app name so that people don't copy and break

---
date: 2023-03-23T11:00:00
title: Highlights for the week ending March 23, 2023
changes_categories:
  - Highlights
owner:
  - https://github.com/orgs/giantswarm/teams/sig-product
---

## General

### Observability

> * prometheus-agent: cluster releases cointains a new component which enables us to monitor your cluster and provides best quality of service. Some cluster upgrade might require manual intervention from our side, especially if you are using prometheus-operator. Feel free to ask team-atlas for any questions or help needed, also more details like why is this needed and which cluster releases are affected can be found in our doc [here](https://docs.giantswarm.io/getting-started/observability/monitoring/prometheus/agent/).

Reviewer 3: @giantswarm/team-teddyfriends just raising awareness on this topic. As customer might experience issue during next cluster upgrade due to us taking ownership of prometheus-operator. Atlas is happy to help during those upgrades.

Reviewer: can we define which cluster release version brings this for each provider plz?

Reviewer 3: This is written in the linked doc https://docs.giantswarm.io/getting-started/observability/monitoring/prometheus/agent

### Observability

> * prometheus-agent: cluster releases cointains a new component which enables us to monitor your cluster and provides best quality of service. Some cluster upgrade might require manual intervention from our side, especially if you are using prometheus-operator. Feel free to ask team-atlas for any questions or help needed, also more details like why is this needed and which cluster releases are affected can be found in our doc [here](https://docs.giantswarm.io/getting-started/observability/monitoring/prometheus/agent/).

Reviewer: ```suggestion
* Prometheus-agent: New cluster releases contain a new component which enables us to monitor your cluster and provides the best quality of service. Some cluster upgrades might require manual intervention from our side, especially if you are using Prometheus Operator. Feel free to ask our team for any questions or help needed (@support-atlas). In case you want to know more details about why we introduced such a change please read our [documentation](https://docs.giantswarm.io/getting-started/observability/monitoring/prometheus/agent/).
```

> * grafana explore: Grafana Explore feature is now enabled. This allow you to execute PromQL (and LogQL later) queries against metrics we collect from your clusters. See https://docs.giantswarm.io/getting-started/observability/grafana/access/ https://grafana.com/docs/grafana/latest/explore/

Reviewer: ```suggestion
* Grafana Explore: It is a new feature enabled in your Grafana installation. This allows you to execute PromQL (and LogQL later) queries against metrics we collect from your clusters. See [our docs for more information](https://docs.giantswarm.io/getting-started/observability/grafana/access/ https://grafana.com/docs/grafana/latest/explore/)
```

> * vertical-pod-autoscaler: v3.x is out which requires to update existing config. Also v3.2.0 improves resources usage as VPA now autoscales itself. Fee free to ask team-atlas for any questions or help needed. See [github.com/vertical-pod-autoscaler@v3.0.0](https://github.com/giantswarm/vertical-pod-autoscaler-app/releases/tag/v3.0.0)

Reviewer: ```suggestion
* Vertical-Pod-Autoscaler (VPA): New version `v.3.x` is our with new improvements. Highlight that now VPA auto-scales itself. This version requires updating the existing config. Read the [migration docs](https://github.com/giantswarm/vertical-pod-autoscaler-app#moving-from-2xx-to-3xx---breaking-change) and feel free to ask our team in case of any doubt (@support-atlas).
```
